### PR TITLE
feat(hyd): A380 -> connect eha actuators on eha elec buss

### PR DESCRIPTION
**Pull request by crocket63**
Fixes #[issue_no]

## Summary of Changes

Update power sources after electrical update 

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs [hunk 1]
```diff
@@ -474,13 +471,13 @@ impl A380AileronFactory {
         let assembly_middle = Self::a380_aileron_assembly(
             context,
             init_drooped_down,
-            Some(Self::MIDDLE_PANEL_EHA_BUS),
+            Some(AC_EHA_BUS),
```
Comment (lukecologne) [inline fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs:474]: The middle aileron is still powered by the AC ESS bus (427XP to be exact, but that is a part of AC ESS)

File: fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs [hunk 2]
```diff
@@ -618,8 +612,7 @@ impl A380SpoilerFactory {
         let spoiler_3 = Self::new_a380_spoiler_element(context, id, 3, None);
         let spoiler_4 = Self::new_a380_spoiler_element(context, id, 4, None);
         let spoiler_5 = Self::new_a380_spoiler_element(context, id, 5, None);
-        let spoiler_6 =
-            Self::new_a380_spoiler_element(context, id, 6, Some(Self::SPOILER_6_EBHA_BUS));
+        let spoiler_6 = Self::new_a380_spoiler_element(context, id, 6, Some(AC_EHA_BUS));
```
Comment (lukecologne) [inline fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs:615]: Same goes for the spoiler 6. Powered by the AC ESS bus.

File: fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs [hunk 3]
```diff
@@ -802,26 +782,10 @@ impl A380ElevatorFactory {
...
-            context,
-            init_drooped_down,
-            if id == ActuatorSide::Left {
-                Some(Self::LEFT_INWARD_PANEL_EHA_BUS)
-            } else {
-                Some(Self::RIGHT_INWARD_PANEL_EHA_BUS)
-            },
-            false,
-        );
+        let assembly_outward =
```
Comment (lukecologne) [inline fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs:785]: Also wrong here: left inner is AC ESS, right inner is AC EHA (correct), left outer is AC EHA (correct), right outer is AC ESS again.

File: fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs [hunk 4]
```diff
@@ -986,20 +940,10 @@ impl A380RudderFactory {
...
-            Self::UPPER_PANEL_LOWER_EBHA_BUS,
-        );
-        let lower_assembly = Self::a380_rudder_assembly(
-            context,
-            init_at_center,
-            false,
-            Self::UPPER_AND_LOWER_PANEL_UPPER_EBHA_BUS,
-            Self::LOWER_PANEL_LOWER_EBHA_BUS,
-        );
+        let upper_assembly =
```
Comment (lukecologne) [inline fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs:943]: Upper rudder upper EBHA is AC ESS, upper rudder lower EBHA is AC EHA, lower rudder upper EBHA is AC ESS, and lower rudder lower EBHA is AC 1

**Review by lukecologne — CHANGES_REQUESTED** [review]
Looks like you assigned every EHA/EBHA the AC EHA bus, which is not correct. Some are still powered by the AC ESS, or even AC 1 busses.

File: fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs (hunk 2 above)
Comment (crocket63) [reply fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs:615]: Mmmm went lost with test ref I used :(.

**Review by 2hwk — APPROVED** [review]

**Pull request merged**